modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java

```diff
@@ -118,7 +117,7 @@ public void testBucketIsFlushableIfEnabled() {
 
                     cluster.buckets().flushBucket(bucketDefinition.getName());
 
-                    await().untilAsserted(() -> assertFalse(collection.exists("foo").exists()));
+                    await().untilAsserted(() -> assertThat(collection.exists("foo").exists()).isFalse());
```

Comment:
Is there a better way to write with Assertj?

Reply:
`collection` is from `couchbase java-client`. I also find it weird but that's what the API offers and TBH don't know much about it `exists("foo")` returns `ExistResult` and `exists()` returns a boolean.

Reply:
oh okay, wasn't aware this is a couchbase type.